dss/dynamodb/__init__.py

```diff
@@ -143,3 +147,31 @@ def delete_item(*, table: str, hash_key: str, sort_key: Optional[str] = None):
     query = {'TableName': table,
              'Key': _format_item(hash_key=hash_key, sort_key=sort_key, value=None)}
     db.delete_item(**query)
+
+
+def update_item(*, table: str, hash_key: str, sort_key: Optional[str] = None, update_expression: Optional[str],
```

Comment:
I think this was already brought up in a prior code review comment, but this should have a test (will follow up and dismiss this comment if there is one in a later PR)

Reply:
There is a test for functionality that uses `update_item` so maybe add a TODO to write a test that tests this function directly. But leave it for a future PR.